setup-ros.sh

```diff
@@ -169,14 +169,10 @@ case ${UBUNTU_VERSION} in
 			setuptools \
 			pyparsing \
 			wheel
-		# RTI Connext
-		DEBIAN_FRONTEND=noninteractive \
-			RTI_NC_LICENSE_ACCEPTED=yes \
-			apt-get install --no-install-recommends --quiet --yes \
-				${RTI_CONNEXT_DDS}
-		# libopensplice69 does not exist on Ubuntu 20.04 and later, so we're attempting to
-		# install it, but won't fail if it does not suceed.
-		apt-get install --no-install-recommends --quiet --yes libopensplice69 || true
```

Comment:
@christophebedard Curious why removed RTI related stuff?

Reply:
This case is now only for Ubuntu 20.04, which is only for Noetic. There are no ROS 2 distros on 20.04 now, so we don't need RTI Connext here.

Reply:
And I removed `libopensplice69`, because, as the comment above says, it does not exist on Ubuntu 20.04 and later, so we don't need it anymore.